### ISSUE TITLE
Is there any plan to migrate to .Net Core 1.0.0 RTM?

### ISSUE THREAD

**Issue by mikeandersun** [Antaris/AspNetCore.Mvc.Widgets#4]
I fork this repo and try to migration. [Here](https://github.com/Antaris/AspNetCore.Mvc.Widgets/blob/develop/src/Antaris.AspNetCore.Mvc.Widgets/Infrastructure/DefaultWidgetInvoker.cs#L101) is a point that i don't know what i do?

```
 var result = await ControllerActionExecutor.ExecuteAsync(executor, widget, arguments);
            var widgetResult = CoerceToWidgetResult(result);
            return widgetResult;
```

It seems  `ControllerActionExecutor.ExecuteAsync` not exists in `1.0.0 RTM`

**Comment by Antaris** [comment]
Yes very much so. I have been working on the RTM upgrade myself internally so will be pushing a new version. Will likely push this to NuGet now too. I'll update this issue when I have pushed the upgrade
